genre.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'item'
+
+class Genre < Item
+  attr_accessor :name, :items
+  attr_reader :id, :publish_date
+
+  def initialize(name)
+    super()
```

Comment:
please note that this class is inherent from item class, so you need to provide all the properties from Genre class to the item using super 

like this example :
 def initialize(args)
    super(args[:genre], args[:author], args[:source], args[:label], args[:publish_date], args[:archived])
    @multiplier = args[:multiplier]
    @last_play_date = args[:last_play_date]
  end

Reply:
It is not necessary that I pass all the properties in the parent class as an argument in super, ruby does not require that. I can just declare the one I want to use when the object is initialized. Also, we do not need source property as we are the only two on the team, it would be necessary if we were four on the team.